task_cli/task.py

```diff
@@ -1,7 +1,7 @@
 import typer
 
 
-def main(name: str = typer.Argument((...))):
+def main(name: str = typer.Argument(' ')):
```

Comment:
It isn't necessary to make the code runs. Just add an argument to the CLI command.

e.g:

task-cli Paulo
Resut: Hello Paulo